layers/b2qt/recipes-core/images/core-image-pelux-qtauto-neptune-dev.bb

```diff
@@ -0,0 +1,12 @@
+#
+#   Copyright (C) 2017 Pelagicore AB
+#   SPDX-License-Identifier: MIT
+#
+
+DESCRIPTION = "Reference PELUX image with QtAuto frontend"
+
+inherit core-image-pelux-qtauto
```

Comment:
Instead `require core-image-pelux-qtauto-neptune.bb`, otherwise this becomes just a duplicate + extra packages.